File: Day-02/26_Remove_Duplicates_from_Sorted_Array.java
import java.util.Arrays;

class Anshu{
    static int duplicate(int test_case[]){
        int k=1;
        for(int i=1 ; i<test_case.length ; i++){
            if(test_case[i] != test_case[i-1]){
                test_case[k] = test_case[i];
                k++;
            }
        }
        return k;
    }
    public static void main(String[] args) {
        int test_case_1[] = new int[]{1,1,2};
		int test_case_2[] = new int[]{0,0,1,1,1,2,2,3,3,4};

		System.out.println("--- test case 1 ---");
        int size_1 = duplicate(test_case_1);
		System.out.print("Input: nums = "+Arrays.toString(test_case_1)+"\nOutput: [");
        for(int i=0 ; i<size_1-1 ; i++){
            System.out.print(test_case_1[i]+", ");
        }
        System.out.println(test_case_1[size_1-1]+"]");

        System.out.println("--- test case 2 ---");
        int size_2 = duplicate(test_case_2);
		System.out.print("Input: nums = "+Arrays.toString(test_case_2)+"\nOutput: [");
        for(int i=0 ; i<size_2-1 ; i++){
            System.out.print(test_case_2[i]+", ");
        }
        System.out.println(test_case_2[size_2-1]+"]");
    }
}

File: Day-02/26_Remove_Duplicates_from_Sorted_Array.py
def duplicate(test_case):
    k = 1
    for i in range(1,len(test_case)):
        if(test_case[i] != test_case[i-1]):
            test_case[k] = test_case[i]
            k+=1
    return k
        
test_case_1 = [1,1,2]
test_case_2 = [0,0,1,1,1,2,2,3,3,4]

print("--- test case 1 ---")
size_1 = duplicate(test_case_1)
print(f"Input: nums = {test_case_1}\nOutput: {test_case_1[0:size_1]}")

print("--- test case 2 ---")
size_2 = duplicate(test_case_2)
print(f"Input: nums = {test_case_2}\nOutput: {test_case_2[0:size_2]}")